src/test/java/com/google/api/codegen/testdata/nodejs_main_library.baseline

```diff
@@ -711,6 +857,23 @@ LibraryServiceApi.prototype.listStrings = function listStrings() {
  * @returns {gax.EventEmitter} - the event emitter to handle the call
  *   status.
  * @throws an error if the RPC is aborted.
+ *
+ * @example
+ *
+ * var api = libraryV1.libraryServiceApi;
+ * var formattedName = LibraryServiceApi.bookPath("[SHELF]", "[BOOK]");
+ * var comment = '';
+ * var stage = Stage.UNSET;
+ * var commentsElement = {
+ *         'comment' : comment,
```

Comment:
The style guide we use for gcloud requires omitting '' for the field name. So this would be `comment: comment,`.
Also this is optional but the indent level is a bit too deep, could be 4-spaces.